# Prevents 'length_error was thrown in -fno-exceptions mode with message "vector"' crash in reverse dependency 'plasma-systemmonitor' on Samsung Galaxy A70 SM-A705FN LineageOS 20 Android 13

--- a/processcore/plugins/CMakeLists.txt
+++ b/processcore/plugins/CMakeLists.txt
@@ -2,4 +2,3 @@ add_definitions(-DTRANSLATION_DOMAIN=\"ksysguard_plugins_process\")
 if (BUILD_NETWORK_PLUGIN)
     add_subdirectory(network)
 endif()
-add_subdirectory(gpu)
